### PR TITLE
rossanodan-issue-33

### PR DESCRIPTION
**Pull request by rossanodan**
## About this pull request

- [x] Added not found component
- [x] Added coming soon component and placeholder test
- [x] Added application router and placeholder test
- [x] Added env file and enabled coming soon mode
- [x] Added PropTypes validation
- [x] Added onClick event to go to homepage button in NotFound

Screenshot 2020-09-21 at 2 35 25 PM
Sep-21-2020 14-51-38

### CODE REVIEW

**Comment by eddiejaoude** [comment]
@rossanodan awesome 👍 

Btw it says `rossanodan requested your review on this pull request.` but also says `it is draft`?

**Comment by rossanodan** [comment]
> @rossanodan awesome 👍
> 
> Btw it says `rossanodan requested your review on this pull request.` but also says `it is draft`?

Yes, draft because I'm adding more things. I will let you know once ready for review 😺

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
```
Comment (eddiejaoude) [inline .gitignore]: `yarn` can probably be removed now?

File: src/feature-comingsoon/comingsoon.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <div className="max-w-md">
+          <div className="text-5xl font-dark font-bold">Coming soon</div>
+            <br />
+            <p className="mb-8">But dont worry, awesome things are on the way.</p>
+      </div>
+    </div>
+  </div>
+);
+
+export default ComingSoon;
```
Comment (eddiejaoude) [inline src/feature-comingsoon/comingsoon.js]: Files need a blank line at the end, GitHub shows a red `-`

**Review by eddiejaoude — DISMISSED** [review]
Looks good, some tiny comments, can be in this PR or the next

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**